### DISCUSSION TITLE
Centralized Penalty List

### DISCUSSION THREAD

**Discussion by bamblebam** [WeMakeDevs/classroom-monitor-bot#52] [Ideas]
So lets create a centralized penalty list for punishing people who repeatedly break rules

**Comment by kaiwalyakoparkar** [comment]
@bamblebam That's awesome. Would you like to describe the issue in detail?

**Reply by bamblebam** [reply]
Yeah

**Comment by bamblebam** [comment]
So there are a lot of rules in our discord server, but right now there is no penalty for breaking any of them. I say we give every member lets say ~300 tardy points and we can keep milestones like if they get below 200 we give them a 1 day ban, if below 100 a 1 week ban and below 0 a perma ban. This can be changed. Though we need some sort of db to store the scores, I dunno if discord has something for this or else we can just use mongo or postgres maybe even firebase.
For each infraction the amount of points deducted should be commensurate. We probably should store them in a db too.
If anyone has any ideas feel free to suggest them.

**Reply by kaiwalyakoparkar** [reply]
Nice thought, Would you like to start working on it?

**Reply by bamblebam** [reply]
No. rn I have internships going on. But I will be happy to guide someone who wants to attempt this.

**Reply by kaiwalyakoparkar** [reply]
Awesome :boom: If someone likes to do it is it okay If I tag you there?

**Reply by bamblebam** [reply]
You can tag me anywhere I will be happy to help

**Reply by rockingrohit9639** [reply]
I would love to work on this issue. Can you @kaiwalyakoparkar and @bamblebam help me out through the development because I am new to discord bot development?

**Reply by Jyoti050402** [reply]
Hey, I would like to take this issue.
